helpers/addresses.py

```diff
@@ -475,6 +475,7 @@
         "BTCb": "0x7130d2A12B9BCbFAe4f2634d864A1Ee1Ce3Ead9c",
         "WBNB": "0xbb4cdb9cbd36b01bd1cbaebf2de08d9173bc095c",
         "CAKE": "0x0E09FaBB73Bd3Ade0a17ECC321fD13a19e81cE82",
+        "EPS": "0xA7f552078dcC247C2684336020c03648500C6d9F",
```

Comment:
for processing this token, already it is on the addr book. [ref](https://github.com/Badger-Finance/badger-multisig/blob/main/helpers/addresses.py#L510). i think after converting this to gas we likely will not touch eps again